react-restaurant/src/Food.js

```diff
@@ -7,25 +7,28 @@ class Food extends React.Component {
     }
 
     componentDidMount() {
-        axios.get(`https://entree-f18.herokuapp.com/v1/menu/12`)
-            .then(response => {
-                console.log(response.data);
-                console.log(response.data.menu_items);
-                console.log(response.data.menu_items[0].description);
-                this.setState({ content: response.data.menu_items });
-                console.log(this.state.content)
-            })
+        if (localStorage.getItem(this.props.menuSection) === null) {
+            axios.get(`https://entree-f18.herokuapp.com/v1/menu/12`)
+                .then(response => {
+                    this.setState({ content: response.data.menu_items });
```

Comment:
Whenever I set state, I think it's helpful for readability to break out the object into multiple lines, similar to how you do it when you declare it at the top of your component. So this line would look like: 

`this.setState({
  content: response.data.menu_items
});`

And the same goes for your set state statement on line 19. Totally preference, but when you get into changing the state of multiple things at once I think it's helpful to break them out into their own lines.